packages/help-center/src/hooks/use-get-zendesk-fields-by-flow.ts

```diff
@@ -0,0 +1,22 @@
+import { isDIFMFlow, isHundredYearDomainFlow, isHundredYearPlanFlow } from '@automattic/onboarding';
+
+export function useGetZendeskFieldsByFlow( flow: string ) {
+	const url = window.location?.href;
+
+	if ( isDIFMFlow( flow ) ) {
+		return {
+			initialMessage: `User is purchasing DIFM plan. ${ url ? ` URL: ${ url }` : '' }`,
+			product: 'DIFM flow',
```

Comment:
@escapemanuele Is there a reason that the product is "DIFM flow" ? From my perspective, this is the flow, not product. 
Also, we should reuse the constant defined in `calypso-product` package instead of defining a new string literal here unless there is a special reason, i.e. `WPCOM_DIFM_LITE`: https://github.com/Automattic/wp-calypso/blob/trunk/packages/calypso-products/src/constants/wpcom.ts#L169

Reply:
You're right. I remove the "product" part, as of pdh1Xd-31k-p2#comment-3006, we'll shift to use a new custom field "messaging_flow" in Zendesk.